plugins/commands/src/AdminCommands.ts

```diff
@@ -108,6 +108,20 @@ const commands: tm.Command[] = [
       ), config.loadmatchsettings.public ? undefined : info.login)
     },
     privilege: config.loadmatchsettings.privilege
+  },
+  {
+    aliases: config.updatemaps.aliases,
+    help: config.updatemaps.help,
+    params: [],
```

Comment:
dont define params when theres none pls..

Reply:
Didn't know that was an option, will fix.